tests/utilities/test_app.py

```diff
@@ -49,16 +58,18 @@ def compose(self) -> ComposeResult:
     def in_running_state(
         self,
         *,
-        initialisation_timeout: float = 0.1,
-    ) -> AsyncContextManager[Capture]:
+        waiting_duration_after_initialisation: float = 0.001,
```

Comment:
Good news! :star2: 
I was (arbitrarily) waiting for 100ms before yielding the app in a "running state", but it turns out that 1ms is enough :rocket:  - which will make most of our integration tests much faster to run! :slightly_smiling_face: 

(expected those that use animation, like some of the test cases of this new test I'm adding in this PR)